mne_bids/read.py

```diff
@@ -88,12 +88,15 @@ def _handle_participants_reading(participants_fname, raw,
         if infokey == 'sex':
             value = _convert_sex_options(infovalue[row_ind],
                                          fro='bids', to='mne')
+            if value is None:
+                value = 0
```

Comment:
After thinking some more about it, we should probably _not set anything_ and emit a warning if we don't know how to map the value to MNE. Because otherwise – if we set `sex = unknown` – it could imply that this information was present in the original BIDS dataset.

```suggestion
            if value is None:  # We don't know how to translate to MNE, so skip.
                warn('Unable to map `sex` value to MNE. Not setting subject sex.')
                continue
```